src/main/java/net/phptravels/gui/pages/HomePage.java

```diff
@@ -141,88 +141,99 @@ public Topbar topbar() {
     }
     
     
-    public void hotelSearchForm(String cityName, String checkin, String checkout, String rooms, String adults, String childs) {
-    	
+    public HotelsSearchResultPage hotelSearchForm(String cityName, String checkin, String checkout, String rooms, String adults, String childs) {
+    	String nationality = "Argentina";
     	hotelDestinationCombobox.click();
     	universalInputField.type(cityName);
     	universalSearchResult.click();
-    	
     	hotelCheckinDate.click();
     	hotelCheckinDate.type(checkin);
     	hotelCheckoutDate.click();
     	hotelCheckoutDate.type(checkout);
-    	
     	hotelTravellersCombobox.click();
     	hotelRoomsInput.type(rooms);
     	hotelAdultsInput.type(adults);
     	hotelCheckinDate.type(childs); 
     	hotelNationalitySelect.click();
-    	hotelNationalitySelect.select("Argentina");
-    	
+    	hotelNationalitySelect.select(nationality);
     	hotelSearchButton.click();
+    	LOGGER.info("[HOTEL]Selected destination " + cityName + " in field destination");
+    	LOGGER.info("[HOTEL]Selected date " + checkin + " in field checkin");
+    	LOGGER.info("[HOTEL]Selected date " + checkout + " in field checkout");
+    	LOGGER.info("[HOTEL]Selected rooms " + rooms + " in field rooms");
+    	LOGGER.info("[HOTEL]Selected adults " + adults + " in field adults");
+    	LOGGER.info("[HOTEL]Selected childs " + childs + " in field childs");
+    	LOGGER.info("[HOTEL]Selected childs " + nationality + " in field childs");
+    	
+    	return HotelsSearchResultPage(driver);
```

Comment:
wrong return, code could not be compiled. `return **new** HotelsSearchResultPage(driver);`